### PR TITLE
Added README.md

### PR DESCRIPTION
**Pull request by bdunogier**

### CODE REVIEW

**Comment by andrerom** [comment]
+1

**Comment by lolautruche** [comment]
+1

**Comment by yannickroger** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# eZ Platform Legacy Bridge
+
+This package integrates eZ Publish Legacy into eZ Platform. It is an extract of the Legacy backwards compatibility 
+features that were part of eZ Publish 5.x.
+
+It is meant to be installed as an addition to eZ Platform, starting from version 2015.03. [How to install and configure
```
Comment (andrerom) [inline README.md]: Post nitpick: should have been 15.01 right? :)

**Pull request merged**